src/components/Person/Person.jsx

```diff
@@ -1 +1,52 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const Person = (props) => {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = props.person;
+
+  const partner = sex === 'f'
+    ? 'husband'
+    : 'wife';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        {`My name is ${name}`}
+      </h2>
+
+      { age && (
+        <>
+          <p className="Person__age">
+            {`I am ${age}`}
+          </p>
+        </>
+      )}
```

Comment:
Your code will be better if you don't use redundant wrapping: <> (
`{age && <p className="Person__age">{`I am ${age}`}</p>}`